lighthouse-core/config/v2/default.json

```diff
@@ -0,0 +1,308 @@
+{
+  "version": 2,
```

Comment:
Agreed we should have verification of the config format but we explicitly are attempting to support two config versions at once to start as we iterate, so I think the version has value as a clue of which path to head toward. A try until it works approach with validation and error messages greatly complicates the MVP. How about `configVersion` until it becomes the default and we nuke the field entirely and rely on validation only?